units/crons.go

```diff
@@ -79,6 +79,10 @@ func StartCrons(ctx context.Context, env cedar.Environment, rpcTLS bool) error {
 
 		return queue.Put(ctx, job)
 	})
+	amboy.IntervalQueueOperation(ctx, remote, time.Hour, time.Now(), opts, func(ctx context.Context, queue amboy.Queue) error {
```

Comment:
did we add a service flag to be able to disable the job, as we see on lines 63-70? Should we check that here?